.github/test/exec.test.js

```diff
@@ -28,4 +28,21 @@ describe("exec", () => {
       expect.objectContaining({ code: "ERR_CHILD_PROCESS_STDIO_MAXBUFFER" }),
     );
   });
+
+  it("buildCmd", () => {
+    const cmd = buildCmd(
+      "foo",
+      // Excluded
+      null,
+      // Arg is *not* trimmed, so trailing whitespace is preserved
+      "spaceAfter ",
+      // Excluded since all-whitespace
+      " \t",
+      // Excluded
+      undefined,
+      // Arg is *not* trimmed, so leading whitespace is preserved
+      " spaceBefore",
+    );
```

Comment:
Perhaps also include numbers, something like: 
```js
buildCmd('create-things', '--num-things', 2)
```

Such a test case would guard against a future edit that drops `0`s 

It's possible things like `NaN` come up but I'm not worried about how they'd be handled.

Reply:
Added tests for every data type I could think of, even the relatively invalid ones.